src/desktop/apps/artist/components/cta.js

```diff
@@ -33,12 +42,44 @@ const send = {
 
 if (sd.ARTIST_PAGE_CTA_ENABLED && sd.ARTIST_PAGE_CTA_ARTIST_ID) {
   metaphysics(send).then(({ artist: artistData }) => {
-    const artist = new Artist(artistData)
-    const view = new ArtistPageCTAView({ artist })
-    $("body").append(view.render().$el)
-    view.initializeMailcheck()
-    setTimeout(() => {
-      view.$el.removeClass("initial")
-    }, 500)
+    const image =
+      artistData.artworks &&
+      artistData.artworks[0] &&
+      artistData.artworks[0].image &&
+      artistData.artworks[0].image.cropped &&
+      artistData.artworks[0].image.cropped.url
+
+    if (sd.ARTIST_PAGE_SIGNUP_MODAL_QA === "experiment") {
```

Comment:
Is there a `split_test.view()` event triggered anywhere? We will need this for the test to work.